docs/book/src/reference/using_an_external_type.md

```diff
@@ -0,0 +1,155 @@
+
+Currently Kubebuilder handles the first two, CRDs and Core Resources, seamlessly.  External CRDs and CRs created via Aggregation must be scaffolded manually.
+
+In order to use a Kubernetes Custom Resource that has been defined in another project
+you will need to have several items of information.
+* The Domain of the CR
+* The Group under the Domain 
+* The Go import path of the CR Type definition.
+
+The Domain and Group variables have been discussed in other parts of the documentation.  The import path would be located in the project that installs the CR.
+
+This document uses `my` and `their` prefixes as a naming convention for repos, groups, and types to clearly distinguish between your own project and the external one you are referencing.
+
+Example external API Aggregation directory structure
+```
+github.com
+    ├── theiruser
+        ├── theirproject
+            ├── apis
+                ├── theirgroup
+                   ├── doc.go
+                   ├── install
+                   │   ├── install.go
+                   ├── v1alpha1
+                   │   ├── doc.go
+                   │   ├── register.go
+                   │   ├── types.go
+                   │   ├── zz_generated.deepcopy.go
+```
+
+In the case above the import path would be `github.com/theiruser/theirproject/apis/theirgroup/v1alpha1`
+
+### Create a project
+
+```shell
+kubebuilder init --domain $APIDOMAIN --owner "MyCompany"
+```
+
+### Add a controller
+
+be sure to answer no when it asks if you would like to create an api? [Y/n]
+```shell
+kubebuilder create api --group mygroup --version $APIVERSION --kind MyKind
+```
+
+## Edit the API files.
+
+### Register your Types
+
+Edit the following file to the pkg/apis directory to append their `AddToScheme` to your `AddToSchemes`:
+
+file: pkg/apis/mytype_addtoscheme.go
+```go
+package apis
+
+import (
+	mygroupv1alpha1 "github.com/myuser/myrepo/apis/mygroup/v1alpha1"
+	theirgroupv1alpha1 "github.com/theiruser/theirproject/apis/theirgroup/v1alpha1"
+)
+
+func init() {
+	// Register the types with the Scheme so the components can map objects 
+	// to GroupVersionKinds and back
+	AddToSchemes = append(
+	  AddToSchemes, 
+	  mygroupv1alpha1.SchemeBuilder.AddToScheme,
+	  theirgroupv1alpha1.SchemeBuilder.AddToScheme,
+	)
+}
+
+```
+
+## Edit the Controller files
+
+### Use the correct imports for your API
+
+file: pkg/controllers/mytype_controller.go
+```go
+package controllers
+
+import (
+	mygroupv1alpha1 "github.com/myuser/myrepo/apis/mygroup/v1alpha1"
+	theirgroupv1alpha1 "github.com/theiruser/theirproject/apis/theirgroup/v1alpha1"
+)
+```
+
+Note that core resources may simply be imported by depending on the API's from upstream Kubernetes:
+
+```go
+package controllers
+// contains core resources like Deployment
+import (
+   v1 "k8s.io/api/apps/v1"
+)
+```
```

Comment:
WDYT about we use the class note here with a header like Kubernetes core types import

Reply:
Not sure if i understand what this is meaning, mind explaining what you want me to do?

Reply:
We can deal with CoreTypes.
See: https://github.com/kubernetes-sigs/kubebuilder/blob/c172069e83d81cc4a6ce67a7a1c3fd2a6e7e2d00/pkg/plugins/golang/options.go#L28-L54

Also, see: 
https://github.com/kubernetes-sigs/kubebuilder/blob/master/testdata/project-v4-multigroup/PROJECT#L86-L90

Reply:
I just verified this on main and I am certain that by default both the PROJECT file is not looking like the testdata file.

When running 
```
go mod init github.com/jakobmoellerdev/testkube
kubebuilder init
[kubebuilder edit --multigroup=true]
kubebuilder create api --group theirgroup --version v1alpha1 --kind ExternalType --controller --resource=false
kubebuilder create api --version v1 --kind Pod --controller --resource=false
```

it always results in the following PROJECT entries:
```
# Code generated by tool. DO NOT EDIT.
# This file is used to track the info used to scaffold your project
# and allow the plugins properly work.
# More info: https://book.kubebuilder.io/reference/project-config.html
domain: my.domain
layout:
- go.kubebuilder.io/v4
[multigroup: true]
projectName: testkube
repo: github.com/jakobmoellerdev/testkube
resources:
- controller: true
  domain: my.domain
  group: theirgroup
  kind: ExternalType
  version: v1alpha1
- controller: true
  domain: my.domain
  kind: Pod
  version: v1
version: "3"
```

What i am referring to is that the api.path element is not set correctly and for core types the `domain` should not be set because this leads to the following generation:

```
//+kubebuilder:rbac:groups=my.domain,resources=pods,verbs=get;list;watch;create;update;patch;delete
//+kubebuilder:rbac:groups=my.domain,resources=pods/status,verbs=get;update;patch
//+kubebuilder:rbac:groups=my.domain,resources=pods/finalizers,verbs=update
```